src/Routes/Default/Default.tsx

```diff
@@ -1,13 +1,32 @@
 import { PageSection } from '@patternfly/react-core';
 import AddWidgetDrawer from '../../Components/WidgetDrawer/WidgetDrawer';
 import GridLayout from '../../Components/DnDLayout/GridLayout';
-import { useAtomValue } from 'jotai';
+import { useAtomValue, useSetAtom } from 'jotai';
 import { lockedLayoutAtom } from '../../state/lockedLayoutAtom';
 import Header from '../../Components/Header/Header';
-import React from 'react';
+import React, { useEffect } from 'react';
+import useCurrentUser from '../../hooks/useCurrentUser';
+import { getWidgetMapping } from '../../api/dashboard-templates';
+import { widgetMappingAtom } from '../../state/widgetMappingAtom';
 
 const DefaultRoute = () => {
   const isLayoutLocked = useAtomValue(lockedLayoutAtom);
+  const setWidgetMapping = useSetAtom(widgetMappingAtom);
+  const { currentToken } = useCurrentUser();
+
+  useEffect(() => {
+    if (!currentToken) {
```

Comment:
What is the token for? If it is auth token, it should always be ready before any UI module is. Otherwise, chrome will not initialize.

Reply:
@Hyperkid123 yes - this is the auth token. On initial render I was seeing it was undefined/empty. I can investigate this in a follow up PR if you'd like.